docs/software/programming.md

```diff
@@ -2,10 +2,12 @@
 
 ## Overview
 
-- [**Python - High-level interpreted programming language**](#python)
-- [**Docker - Build, ship, and run distributed applications**](#docker)
-- [**Docker Compose - Run multi-container applications on Docker**](#docker-compose)
-- [**Portainer - Docker container management**](#portainer)
+  - [Python](#python)
+  - [Docker -  Create, deploy, and run applications using containers](#docker)
+  - [Docker Compose - Define and run multi-container Docker applications](#docker-compose)
+  - [Portainer - Lightweight management UI, managing your Docker host or Swarm cluster](#portainer)
```

Comment:
Let's stick with the way this overviews are defined on all other pages (**fat** link text), or otherwise align them on all other pages. Also the Docker description got removed?

The two space indentation is obsolete here.

Reply:
I will update the links (**fat** link text) - it's good :)

Reply:
I didn't compare or check which one looks better, but I think it makes sense to stick with the visual conventions we applied to all software docs pages or otherwise change them everywhere 🙂.